bucket/lightbulb.json

```
@@ -2,28 +2,10 @@
     "version": "1.6.4.1",
     "description": "A program that reduces eyestrain produced by staring at a computer screen when working late hours",
```

This is completely useless description.

```suggestion
    "description": "Continuously adjusts gamma, transitioning the display color temperature from cold blue to warm yellow.",
```